build.proj

```diff
@@ -66,28 +66,31 @@
     <DelaySign Condition =" '$(DelaySign)' == '' ">false</DelaySign>
     <Scope Condition=" '$(Scope)' == '' ">all</Scope>
   </PropertyGroup>
- 
+
   <ItemGroup>
     <LibraryFxTargetList Condition=" '$(FxTargetList)' == '' and '$(Scope)' == 'ClientRuntime' " Include="portable;net45" />
     <LibraryFxTargetList Condition=" '$(FxTargetList)' == '' and '$(Scope)' == 'all' " Include="portable;net45" />
     <LibraryFxTargetList Condition=" '$(FxTargetList)' == '' and '$(Scope)' == 'CodeGenerator' " Include="net45" />
     <LibraryFxTargetList Condition=" '$(FxTargetList)' != '' " Include="$(FxTargetList)" />
   </ItemGroup>
-  
+
   <PropertyGroup>
-    <NuGetCommand>&quot;$(LibraryToolsFolder)\nuget.exe&quot;</NuGetCommand>
-  </PropertyGroup>
+    <NuGetCommand Condition=" '$(OS)' == 'Windows_NT'">&quot;$(LibraryToolsFolder)\nuget.exe&quot;</NuGetCommand>
+    <NuGetCommand Condition=" '$(OS)' != 'Windows_NT'">mono &quot;$(LibraryToolsFolder)\nuget.exe&quot;</NuGetCommand>
+    <NuGetSources>https://www.myget.org/F/xunit;https://www.nuget.org/api/v2/</NuGetSources>
```

Comment:
If this source is only for command line build, that is fine. Otherwise can we use the xunit package from the official nuget source? I hope we can avoid any non nuget official sources, for the main reason that we will have to document across to let people add this source. Otherwise people will see the build fail at first try and then just send a mail to us for why.